src/pages/HomePage.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import Home from '../features/Home';
```

Comment:
We could use absolute import here, wdyt ?

Reply:
done